apps/system/test/unit/advanced_telemetry_test.js

```diff
@@ -478,9 +524,9 @@ suite('AdvancedTelemetry:', function() {
         assert.equal(req.clientId, wrapper.clientId);
 
         // Verify the application object picked up the settings correctly.
-        assert.deepEqual(req.application, wrapper.application);
-        // Verify that the Histograms are intact.
-        assert.deepEqual(req.payload, wrapper.payload);
+        assert.deepEqual(req.application, packedWrapper.application);
+        // Verify that the Histograms are intact and are properly packed.
+        assert.deepEqual(req.payload, packedWrapper.payload);
```

Comment:
Is the old format `wrapper` with `ranges` and `counts` on line 313 still necessary? it seems like `packedWrapper` can safely supersede it.